modules/virt-resources-monitored-actively-in-Virtualization-Overview-page.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-resources-monitored-actively-in-Virtualization-Overview-page.adoc
+//
+
+[id="virt-resources-monitored-actively-in-Virtualization-Overview-page_{context}"]
+= Resources monitored actively in the Virtualization Overview page
```

Comment:
As I mentioned in my other comment, the title and column headings of this table are very similar to the other table in this assembly. This might cause confusion. I would suggest changing the table title to "Virtualization Overview page metrics".

Reply:
How about: Monitored resources, fields, and metrics. I chose this heading because "permissions" does not fall under metrics.

Reply:
There are plans to add additional monitored items to the Virtualization Overview page that may not fall under metrics.